cloudcomposerdiff/README.md

```diff
@@ -0,0 +1,184 @@
+# A python command line tool to diff two Cloud composer environments.
+
+## Running this code
+
+### Clone repository
+
+Clone the code from GitHub to your local machine
+
+### Install locally with pip
+
+```
+pip install cloudcomposerdiff
+```
+
+### Authorizing the application using gcloud
+
+Install [gcloud](https://cloud.google.com/sdk/gcloud)
+
+Select a user which has access to both composer environments to compare.
+
+Wit this user account, initialize gcloud:
+
+```
+gcloud init
+```
+
+Next, authorize the application to access cloud composer environments:
+
+```
+gcloud auth application-default login
+```
+
+### Use the appplication
+
+```
+cloudcomposerdiff \
+--env1_project_id <> \
+--env1_location <> \
+--env1_name <> \
+--env2_project_id <> \
+--env2_location <> \
+--env2_name <>
+```
+
+## Developing this application
+
+### Clone repository
+
+Clone the code from GitHub to your local machine
+
+### Getting the correct version of python with pyenv
+
+Install [pyenv](https://github.com/pyenv/pyenv).
+
+Use pyenv to install a suitable version of python, e.g. 3.10.0
+```
+$ pyenv install 3.10.0    
+```
+
+Use pyenv to switch to python 3.10.0
+```
+$ pyenv shell 3.10.0
+```
+
+### Create a virtual environment with venv module
+
+```
+python3 -m venv venv
```

Comment:
```suggestion
python -m venv venv
```

If you're using pyenv, I believe it's okay to use `python` here instead of `python3`. I try to avoid using `python3` because it implies that `python2` is around and I don't want to give anyone any ideas about using it 😁

Reply:
ah ok, thanks for the feedback. 

Yes, agree. Let's not bring up python2 ! 

I have changed all references from python3 -> python.